apps/code-review/src/app/pages/overview/overview.component.ts

```diff
@@ -65,6 +75,43 @@ export class OverviewComponent {
         this.isSidenavOpened.update((state) => !state);
     }
 
+    public setFileFromQuery(): void {
+        setTimeout(() => {
+            if (this.file) {
+                const filePath = this.file.split('/');
+                const files = this.findFilesByPathArray(filePath);
```

Comment:
```suggestion
                const entities: ProjectEntities[] = this.findFilesByPathArray(filePath);
```

Явно вказуючи тип, ми уточнюємо що саме за entities тут